app/src/organisms/LabwareDetails/StyledComponents/__tests__/ExpandingTitle.test.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react'
+import { renderWithProviders } from '@opentrons/components'
+import { ExpandingTitle } from '../ExpandingTitle'
+import { getFootprintDiagram } from '../../measurementGuide'
+import { fireEvent } from '@testing-library/react'
```

Comment:
https://opentrons.atlassian.net/wiki/spaces/RPDO/pages/3265364497/Code+Cook+Book+for+App#import-statement

I guess most of us put the component that we want to test as the last.

```suggestion
import * as React from 'react'
import { fireEvent } from '@testing-library/react'
import { renderWithProviders } from '@opentrons/components'
import { getFootprintDiagram } from '../../measurementGuide'
import { ExpandingTitle } from '../ExpandingTitle'
```